flash/vision/classification/transforms.py

```diff
@@ -38,7 +40,6 @@ def default_train_transforms(image_size: Tuple[int, int]) -> Dict[str, Callable]
             "post_tensor_transform": ApplyToKeys(
                 DefaultDataKeys.INPUT,
                 K.geometry.Resize(image_size),
-                K.augmentation.RandomHorizontalFlip(),
```

Comment:
we need to run a retrain if we change this and update default weights in the examples

Reply:
Haven't changed it, it just gets added in line 70